posts/2024-06-20-types-team-update.md

```diff
@@ -0,0 +1,225 @@
+
+We also organized another in-person Types Team meetup last October, immediately prior to EuroRust. We discussed the state of the team, looked at current implementation challenges and in-progress work, and reviewed and updated [the roadmap from the previous meetup][PrevRoadmap]. Most of this will be covered in this blog post.
+
+Finally, as discussed in the [RFC](https://rust-lang.github.io/rfcs/3254-types-team.html), we would like to have leads rotate out regularly, largely to help share the burden and experience of leads' work. So with that being said, [@nikomatsakis](https://github.com/nikomatsakis) is rotating out and [@lcnr](https://github.com/lcnr) is joining to co-lead alongside [@jackh726](https://github.com/jackh726/).
+
+## Roadmap progress and major milestones
+
+### The next-generation trait solver
+
+There has been [a lot of work][NewSolver] on the [next-generation trait solver][InitiativeRepo].
+The initiative posted [a separate update][InitiativeUpdate] at the end of last year. While
+we would have liked to [stabilize its use in coherence][StabilizeNS] a few months ago,
+this surfaced additional small behavior regressions and hangs, causing delays. We are working on fixing these issues and intend to merge the stabilization PR soon. We are getting close to compiling the standard library
+and the compiler with the new solver enabled everywhere, after which will be able to run
+crater to figure out the remaining issues. We expect there to be a long tail of minor issues
+and behaviorial differences from the existing implementation, so there's still a lot to do
+here. There are also open design questions which we will have to resolve before stabilizing
+the new implementation.
+
+### Async and `impl Trait`
+
+We stabilized `async`-fn in traits (AFIT) and return-position `impl Trait` in
+traits (RPTIIT) in version 1.79 thanks to a significant effort by [@compiler-errors] and
+[@spastorino]. [@cjgillot] greatly improved the way generators, and therefore async functions,
+are represented in the type system[^107421]. This allowed us to support recursive
+`async`-functions without too much additional work[^117703].
+
+Designing the next-generation trait solver surfaced issues and future-compatability challenges
+of our type-alias `impl Trait` (TAIT) implementation using the old trait solver. We are
+currently reworking the design and implementation. [@oli-obk] is spear-heading this effort.
+This also impacts RPIT edge-cases, forcing us to be careful to avoid accidental breakage.
+There have also been some new language design questions for TAIT, so we will likely first
+stabilize associated type position `impl Trait` (ATPIT) as it avoids these language design
+issues while still providing the same capabilities.
```

Comment:
```suggestion
stabilize associated type position `impl Trait` (ATPIT) as it avoids these
questions while still closing the expressiveness gap.
```

I'd suggest weakening the language from "issues" to "questions" (as was done earlier in the sentence), removing some repeated verbiage, and saying that ATPIT closes the expressiveness gap rather than offering the same "capabilities".

(Maybe it does offer the same capabilities, but it's more obviously true that it closes the expressiveness gap.)